src/components/timeline/TrackContextMenu.js

```diff
@@ -488,7 +488,7 @@ class TimelineTrackContextMenu extends PureComponent<Props> {
 
   render() {
     const { globalTrackOrder, globalTracks } = this.props;
-
+    const rightClickedTrack = this.props.rightClickedTrack;
```

Comment:
`rightClickedTrack` can be place in the `this.props` destructure right above it, on line 490.

```
const { globalTrackOrder, globalTracks, rightClickedTrack } = this.props;
```

Reply:
Yeah my bad , i should've done that . I have fixed it